### ISSUE TITLE
Core error with Resharper file

### ISSUE THREAD

**Issue by OsirisTerje** [editorconfig/editorconfig-visualstudio#33]
From time to time, Editorconfig crashes with a core error on files named like StackTractExplorerEditor.Bozepih , this extension may vary.  The file comes from Resharper, but I am not able to see it afterwards.  It seems to be a temporary file.

![image](https://cloud.githubusercontent.com/assets/203432/12025733/ffb4664e-adb1-11e5-80d9-5851c2d53c2d.png)

**Comment by peteraritchie** [comment]
dup of https://github.com/editorconfig/editorconfig/issues/167 ?

**Comment by oysteinkrog** [comment]
I see this too

**Comment by peteraritchie** [comment]
Was this closed as fixed or closed as dup?  Because the dup seems to be closed as well, I don't think the issue has been addressed?

**Comment by zxxc** [comment]
![image](https://cloud.githubusercontent.com/assets/13364984/21503114/d1f69f0a-cc65-11e6-87c3-b5282b67a693.png)

**Comment by joeynovak** [comment]
I still get it as well.

**Comment by peteraritchie** [comment]
yeah, switched to VS 2017 that doesn't need this add-on...  ¯\_(ツ)_/¯
